cmake/external/boringssl.cmake

```diff
@@ -21,14 +21,14 @@ endif()
 set(patch_file 
   ${CMAKE_CURRENT_LIST_DIR}/../../scripts/git/patches/boringssl/0001-disable-C4255-converting-empty-params-to-void.patch)
 
-set(commit_tag 83da28a68f32023fd3b95a8ae94991a07b1f6c62)
+set(boringssl_commit_tag 83da28a68f32023fd3b95a8ae94991a07b1f6c62)
```

Comment:
Just curious, why do these need to be changed?

Reply:
I had mistakenly typoed the commit tag in the applicaiton of it, and it was still set to the boring ssl commit tag from the previous cmake file, which had a nasty `silent` error that the commit couldn't be found (but the build kept on going).  Having an explicity variable for each checkout will make things easier to debug later, **I think**.